### PR TITLE
Steak out panel redesign

### PR DESCRIPTION
**Pull request by VitorVieiraZ**
- ```StakeoutPanel.qml``` redesigned to ```MMStakeoutPanel.qml```.

### CODE REVIEW

File: app/images/gpsDirectionIcon.svg
Comment (PeterPetrik) [inline app/images/gpsDirectionIcon.svg:1]: not used, delete

File: app/qml/CMakeLists.txt [hunk 1]
```diff
@@ -156,6 +156,31 @@ set(MM_QML
...
+    ManageInvitationsPage.qml
+    Notification.qml
+    NotificationBanner.qml
+    NumberSpin.qml
+    PanelItem.qml
+    PanelTabButton.qml
+    ProjectLimitDialog.qml
+    RegistrationFinishPage.qml
+    RoundIndicator.qml
+    SearchBar.qml
```
Comment (PeterPetrik) [inline app/qml/CMakeLists.txt:179]: remove

File: app/qml/CMakeLists.txt [hunk 2]
```diff
@@ -156,6 +156,31 @@ set(MM_QML
...
+    MMStakeoutPanel.qml
+    StorageLimitDialog.qml
+    SwitchWorkspacePage.qml
+    WhatsNewDialog.qml
```
Comment (PeterPetrik) [inline app/qml/CMakeLists.txt:183]: remove

File: app/qml/MMStakeoutPanel.qml [hunk 3]
```diff
@@ -0,0 +1,324 @@
+/***************************************************************************
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ ***************************************************************************/
+
+import QtQuick
```
Comment (PeterPetrik) [inline app/qml/MMStakeoutPanel.qml:10]: move the file to `gps/` folder

File: app/qml/MMStakeoutPanel.qml [hunk 4]
```diff
@@ -0,0 +1,324 @@
...
+    MMHeader {
+      id: header
+
+      backVisible: false
+
+      title: qsTr("Stake out")
+      titleFont: __style.t2
+
+      MMRoundButton {
+        id: backBtn
```
Comment (PeterPetrik) [inline app/qml/MMStakeoutPanel.qml:130]: closeBtn

File: app/qml/MMStakeoutPanel.qml [hunk 5]
```diff
@@ -0,0 +1,324 @@
...
+    id: roundedRect
+
+    anchors.fill: parent
+    color: __style.whiteColor
+
+    MMHeader {
+      id: header
+
+      backVisible: false
+
```
Comment (PeterPetrik) [inline app/qml/MMStakeoutPanel.qml:125]: rightMarginShift: closeBtn.width + __style.pageMargins

File: app/qml/MMStakeoutPanel.qml [hunk 6]
```diff
@@ -0,0 +1,324 @@
...
+      anchors.top: header.bottom
+      anchors.leftMargin: __style.pageMargins
+      anchors.rightMargin: __style.pageMargins
+      anchors.topMargin: __style.pageMargins
+
+      spacing: __style.margin12
+
+      Row {
+        width: parent.width
+        height: 67 * __dp
```
Comment (PeterPetrik) [inline app/qml/MMStakeoutPanel.qml:161]: __style.row67

File: app/qml/MMStakeoutPanel.qml [hunk 7]
```diff
@@ -0,0 +1,324 @@
...
+
+        MMGpsDataText{
+          titleText: qsTr( "Feature" )
+          descriptionText: root.targetPair ? __inputUtils.featureTitle( root.targetPair, __activeProject.qgsProject ) : ""
+        }
+
+        MMGpsDataText{
+          titleText: qsTr( "Distance" )
+          descriptionText: remainingDistance >= 0 ?__inputUtils.formatDistanceInProjectUnit( remainingDistance, 2 ) : "N/A"
```
Comment (PeterPetrik) [inline app/qml/MMStakeoutPanel.qml:170]: qsTr("N/A")

File: app/qml/MMStakeoutPanel.qml [hunk 8]
```diff
@@ -0,0 +1,324 @@
...
+            Shape {
+              id: rootShape
+
+              property real centerX: width / 2
+              property real centerY: height / 2
+
+              anchors.fill: parent
+
+              ShapePath {
+                strokeColor: closeRangeModeComponent.state === "notAtTarget" ? InputStyle.labelColor : InputStyle.fontColorBright
```
Comment (PeterPetrik) [inline app/qml/MMStakeoutPanel.qml:225]: InputStyle cannot be used anywhere

File: app/icons/icons.qrc [hunk 9]
```diff
@@ -87,5 +87,6 @@
         <file>XTwitter.svg</file>
         <file>Youtube.svg</file>
         <file>ZoomToProject.svg</file>
+        <file>GpsDirection.svg</file>
```
Comment (PeterPetrik) [inline app/icons/icons.qrc:90]: revert you do not need it. This is always composed from 
1/ direction image (triangle where you look)
2/ marker (dot in middle)
3/ light circle (dynamic - that shows your GPS accuracy - e.g. if you GPS has 10 cm accuracy your green circle is 10cm wide)

File: app/icons/GpsDirection.svg
Comment (PeterPetrik) [inline app/icons/GpsDirection.svg:1]: remove

File: app/mmstyle.h [hunk 10]
```diff
@@ -114,6 +114,7 @@ class MMStyle: public QObject
     Q_PROPERTY( QUrl globeIcon READ globeIcon CONSTANT )
     Q_PROPERTY( QUrl globalIcon READ globalIcon CONSTANT )
     Q_PROPERTY( QUrl gpsIcon READ gpsIcon CONSTANT )
+    Q_PROPERTY( QUrl gpsDirectionIcon READ gpsDirectionIcon CONSTANT )
```
Comment (PeterPetrik) [inline app/mmstyle.h:117]: remove

File: app/mmstyle.h [hunk 11]
```diff
@@ -328,6 +329,7 @@ class MMStyle: public QObject
     QUrl globeIcon() {return QUrl( "qrc:/Globe.svg" );}
     QUrl globalIcon() {return QUrl( "qrc:/Global.svg" );}
     QUrl gpsIcon() {return QUrl( "qrc:/GPSIcon.svg" );}
+    QUrl gpsDirectionIcon() {return QUrl( "qrc:/GpsDirection.svg" );}
```
Comment (PeterPetrik) [inline app/mmstyle.h:332]: remove

File: app/qml/MMStakeoutPanel.qml [hunk 12]
```diff
@@ -0,0 +1,324 @@
...
+              id: positionMarker
+
+              PositionDirection {
+                id: positionDirection
+
+                positionKit: __positionKit
+                compass: Compass { id: ccompass }
+              }
+
+              Image {
```
Comment (PeterPetrik) [inline app/qml/MMStakeoutPanel.qml:286]: See MMPositionMarker, maybe you can use it instead of this Image at all?

**Comment by PeterPetrik** [comment]
Fix build, rebase to current master, squash all commits to 1 commit

File: app/qml/form/MMPreviewPanel.qml [hunk 13]
```diff
@@ -198,7 +198,7 @@ Item {
             buttonText: qsTr( "Stake out" )
             iconSource: __style.positionTrackingIcon // TODO: change to stakeout icon
```
Comment (PeterPetrik) [inline app/qml/form/MMPreviewPanel.qml:199]: Change icon to stakeout icon

**Comment by VitorVieiraZ** [comment]
- gpsDirectionIcon removed and replaced for MMPositionMarker
- stakeOutIcon added and in gallery Icons Page
- adjustments done

**Review by PeterPetrik — COMMENTED** [review]
looks good

**Pull request merged**